.github/workflows/flutter-prep.yaml

```diff
@@ -13,13 +13,18 @@ on:
       needs-checkout-merge:
         description: 'Whether the PR should be merged during the checkout step. Necessary for pull_request_target workflows.'
         type: boolean
+      requires-label:
+        description: 'Specifies the label required for the workflow to run. If not provided, the workflow will always run.'
```

Comment:
should this say "If not provided, the workflow will not run." ?

Reply:
Nope, it will always run unless a label is required to run.